src/pymovements/measure/measures.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2024 The pymovements Project Authors
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+"""Provides eye movement measure implementations."""
+from __future__ import annotations
+
+import polars as pl
+
+from pymovements.measure.library import register_measure
+
+
+@register_measure
+def null_ratio(column: str, column_dtype: pl.DataType) -> pl.Expr:
```

Comment:
can the dtype be inferred? e.g. 

```python
if column_dtype is None:
    column_dtype = pl.col(column).dtype
```

?

Reply:
Unfortunately an expression has no dtype and there's no expression for getting the dtype (yet).

I currently don't know how to solve this more elegantly..

Here's the error:

```python
pl.col('a').dtype
Traceback (most recent call last):
  File "/home/krakowczyk/workspace/pymovements/venv/lib/python3.11/site-packages/IPython/core/interactiveshell.py", line 3505, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)
  File "<ipython-input-7-1b8e72eae2a9>", line 1, in <module>
    pl.col('a').dtype
AttributeError: 'Expr' object has no attribute 'dtype'
```